keras-dreambooth-sprint/README.md

```diff
@@ -1,95 +1,135 @@
+
+## Submission Categories
+
+There are 3 different categories in which you can participate:
+- Nature and Animals (`nature`)
+- Sci-fi/Fantasy Universes (`sci-fi`)
+- Consentful (`consentful`): Partner up with an artist to fine-tune on their style, with their consent! Make sure to include a reference to the artist’s express consent (e.g. a tweet) in your model card.
+- Wild Card (`wild-card`): If your submission belongs to any category that is not above, feel free to tag it with wild-card so we can evaluate it out of that category.
 
-[Link to notebook](https://colab.research.google.com/drive/1Bh49YBYgaNc3X5aRzaMZNska4ErvU3lJ?usp=sharing) 
+Add the category with their IDs to the model cards for submission and add `keras-dreambooth` to model card metadata in tags section. Here's an example [model card](https://huggingface.co/spaces/keras-dreambooth/example-submission/blob/main/README.md). All the submissions will be populated [in this leaderboard](https://huggingface.co/spaces/keras-dreambooth/leaderboard) and ranked according to likes on a given Space to determine the winners.
 
-You can fine-tune on any concept that you want. Couple of inspirations for you: 
+Couple of inspirations for you: 
 
+TODO: it might be confusing to show examples not using keras. maybe worth for us to build some of the examples and share them
 1. Lowpoly World: This [model](https://huggingface.co/MirageML/lowpoly-world) generates lowpoly worlds 🤯🌍
 2. Future Diffusion: This [model](https://huggingface.co/nitrosocke/Future-Diffusion) generates images in futuristic sci-fi concepts 🤖
 3. Fantasy sword: This [model](https://huggingface.co/MirageML/fantasy-sword) generates swords for fantasy themed games 🧙‍♂️
 
-Are you looking to stand out from the crowd? If you partner up with an artist to fine-tune on a **********consentful********** concept, you’ll be eligible to submit your model under the `consentful` theme!
-
-If you need more pointers on Dreambooth implementation with Keras, you can check out [this repository](https://github.com/sayakpaul/dreambooth-keras). 
+## Compute
 
 [ADD DETAILS ABOUT COMPUTE SPONSORSHIP]
 
-### Dreambooth Diffusers Integration with KerasCV
+## Model Training
 
-As of now, inference and deployment options of `KerasCV` is limited, which `diffusers` library could come for a rescue. With only few lines of code, we can convert a `KerasCV` model into a `diffusers` one and use `diffusers`’ pipelines to perform inference. You can get more information [here](https://huggingface.co/docs/diffusers/main/en/using-diffusers/kerascv). Also check out [this Space](https://huggingface.co/spaces/sayakpaul/convert-kerascv-sd-diffusers) for converting your `KerasCV` model to `diffusers`one.
+You can find the notebook here and adjust it according to your own dataset 👇  
 
-```
-from diffusers import StableDiffusionPipeline
+TODO: The notebook below is a Google Drive. We should add it to the GitHub repo
```

Comment:
I think it's okay for now.

Reply:
I think having it in the repo for the event would be useful, that way others can open suggestions, fix typos, etc. As an example done from previous event: https://github.com/huggingface/community-events/tree/main/whisper-fine-tuning-event